api/server/server.go

```diff
@@ -39,7 +39,7 @@ import (
 )
 
 var (
-	activationLock chan struct{}
+	activationLock chan struct{} = make(chan struct{})
```

Comment:
Don't wanna touch @ibuildthecloud commit :) Let's create issue for contributors about this later.